server/notebooks/views.py

```diff
@@ -28,9 +28,6 @@ def notebook_view(request, pk):
 
 @login_required
 def new_notebook_view(request):
-    template = loader.get_template('notebook.html')
-    return HttpResponse(
-        template.render({
-            'user_info': _get_user_info_json(request.user),
-            'jsmd': ''
-        }, request))
+    notebook = Notebook.objects.create(owner=request.user)
+    NotebookRevision.objects.create(notebook=notebook)
+    return redirect('notebook-view', pk=notebook.id)
```

Comment:
`/new/` is currently wired to create the notebook when the user decides to manually save:

https://github.com/iodide-project/iodide/blob/b4b0a06ef634cc949575d561c158ccabb7b5c9ba/src/actions/actions.js#L314

I think it would be best to leave this change out for now. Eventually we might want to do something more similar to what github does when creating a new repository (i.e. select a name + template in a form, then proceed to actually create the notebook): https://github.com/new